src/components/pages/Site/Site.js

```diff
@@ -82,13 +82,26 @@ const ReadOnlySiteContent = ({
 }
 
 const enforceNumberInput = (event) => {
-  const numbersPlusMinusAndDotSymbols = /[0-9+-.]/.source
-  const regex = new RegExp(
-    `${numbersPlusMinusAndDotSymbols}|(Backspace|Tab|Delete|ArrowLeft|ArrowRight|ArrowUp|ArrowDown)`,
-  )
-  const preventKeyPressNotMatchInRegex = !event.key.match(regex) && event.preventDefault()
+  const specialActionAndCharacterKeys = [
```

Comment:
I think some comments here are warranted that we are doing all of this to allow copy paste shortcuts.

Reply:
Other than that looks good. With comments, considered approved.